charts/testkube-enterprise/values.yaml

```diff
@@ -48,6 +48,21 @@ global:
     issuer: ""
   # -- Common labels which will be added to all resources
   labels: &common_labels {}
+  mongo: &global_mongo_config
+    # -- Mongo DSN connection string
+    dsn: "mongodb://testkube-enterprise-mongodb:27017"
+    # -- Mongo DSN connection string secret ref (secret must contain key MONGO_DSN) (default is `mongo-dsn`)
+    dsnSecretRef: ""
+    # -- Mongo database name
+    database: "testkubeEnterpriseDB"
+    # -- Mongo read preference (primary|primaryPreferred|secondary|secondaryPreferred|nearest)
+    readPreference: secondaryPreferred
```

Comment:
what is secondaryPreferred?

Reply:
by default, all operations go to master, this suggests the mongo client to read from a read replica if possible, if not, read from the master replica (in case of a single mongo node like in-cluster deployment)